pages/blog/_.vue

```diff
@@ -98,10 +98,16 @@
             error({ statusCode: 404, message: app.i18n.t('messages.notFound') });
             return;
           }
-          store.commit('breadcrumb/setBreadcrumb', {
-            text: response.items[0].fields.name,
-            active: true
-          });
+          store.commit('breadcrumb/setBreadcrumbs', [
+            {
+              text:  app.i18n.t('blog.blog'),
+              href: '/blog'
```

Comment:
Use `to` and make this localisation-aware.

Reply:
I've used to and added a TODO as there's also no named route for blogs yet.

Reply:
Right you are.